group_vars/all/domains.yml

```diff
@@ -0,0 +1,17 @@
+# URL
```

Comment:
what the purpose of this file ?

Reply:
Defining default values for domain related variables.